tests/e2e/utils/requestHandlers/tokens/CheMultiuserTokenHandler.ts

```diff
@@ -37,4 +37,16 @@ export class CheMultiuserTokenHandler implements ITokenHandler {
         }
 
     }
+
+    /**
+     * Append `/` if it's not in the URL yet. Keycloak doesn't handle double `//` in URL well.
```

Comment:
Could you please clarify what exactly this method does?
Because it is a bit understandable for me.
Or show an example.

Reply:
If `keycloakUrl` ends with `/` it does nothing.
If `keycloakUrl` does not end with `/`, it adds it.

This is because before these changes, when `TS_SELENIUM_BASE_URL` ended with `/`, resulting keycloakUrl (on line 23) was in format `<keycloakUrl>//auth/realms...` (notice the double `//` in the URL) and keycloak was not able to handle this correctly.
The changes in this file are to ensure that the resulting keycloak URL will not have the double forward slashes.

Reply:
Ok, I see, thank you for responding.
Could it be also helpful for other environment variables which provide URL's ?

I don't push you to implement this for the current PR, I just suggest create an issue for that.

Reply:
I didn't see this problem with other URL's. I would create issue if I encounter this on some other place as well. Thanks ;-)

Reply:
For example, if you provide the "TS_SELENIUM_BASE_URL" with the "/" at the end it will leads to the error in the "HappyPath" test which opens workspace by direct URL which includes also this variable value.